src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/integration/PniIntegrationTest.kt

```diff
@@ -51,9 +51,9 @@ class PniIntegrationTest :
     assessmentId = 2114584,
     programmePathway = "MISSING_INFORMATION",
     needsScore = NeedsScore(
-      overallNeedsScore = 6,
+      overallNeedsScore = 4,
       basicSkillsScore = 33,
-      classification = "HIGH_NEED",
+      classification = "MEDIUM_NEED",
```

Comment:
Might be worth checking if this needs to say "INFORMATION_MISSING"

Reply:
I can't see anything in the UI or API tickets to suggest this, although I'll check.